flink-shaded-zookeeper-parent/flink-shaded-zookeeper-35/pom.xml

```diff
@@ -34,7 +34,7 @@ under the License.
 
     <properties>
         <zookeeper.version>3.5.10</zookeeper.version>
-        <curator.version>5.3.0</curator.version>
```

Comment:
The commit message for the curator update is wrong (we update from 5.3 to 5.4)

Reply:
Fixed